01-JS-DOM-APIs/1-2.js

```diff
@@ -0,0 +1,24 @@
+function Matrix(){
+  matrix = [
+    ["Esteban Quito","Ecuador", "North Korea"],
+    ["John Smith","Great Britain", "Japan"],
+    ["Federico Lopez","Spain", "Australia"],
+  ];
+  console.log(matrix[0][0]);
+  return matrix;
+}
+
+function importMatrix(matrix) {
+  console.log(matrix);
+
+  for(var indexTR = 0; indexTR < 3; indexTR++ ){
+    var trNode = document.createElement("tr");
```

Comment:
The guide says we should always use let instead var. Also every variable declaration should go outside loop in order to make this function more perfomant and avoid declaration several times.